actions/overlay_action.go

```diff
@@ -1,3 +1,26 @@
+/*
+Overlay Action
+
+Recursive copy of directory or file to target root filesystem.
+
+Yaml syntax:
+ - action: overlay
+   origin: name
+   source: directory
+   destination: directory
+
+Mandatory properties:
+
+- origin -- reference to named file or directory.
+
+Optional properties:
+
+- source -- relative path to the directory or file located in path referenced by `origin`.
+In case if this property is absent then pure path referenced by 'origin' will be used.
+
+- destination -- absolute path in the target rootfs there 'source' will be copied.
+All existing files will be overwritten. Root ('/') of target rootfs will be used in case of property absence.
```

Comment:
If desitnation isn't set / of the rootfs ... Using 'target rootfs' is odd as there is only one rootfs :) Also i just notice this now but we keep referring to rootfs, but the origin name is "filesystem" So make better to keep that jargon in the documentation as well

Reply:
ok